Only use InitialMasterNodes during bootstrap (#85929)
elastic/elasticsearch: docs/reference/modules/discovery/bootstrapping.asciidoc

When you start a master-eligible node, you can provide this setting on the
command line or in the `elasticsearch.yml` file. After the cluster has formed,

> this setting should no longer be used. It should not be set for master-ineligible

Reviewer: I think that's still weaker than it could be. How about making it a command to the user, like this?

```suggestion
remove this setting from the configuration. It should not be set for master-ineligible
```
